x-pack/plugins/apm/public/components/alerting/TransactionDurationAlertTrigger/index.tsx

```diff
@@ -57,7 +72,7 @@ export function TransactionDurationAlertTrigger(props: Props) {
     aggregationType: 'avg',
     windowSize: 5,
     windowUnit: 'm',
-    transactionType: transactionTypes[0],
+    transactionType,
```

Comment:
This is not available on errors & metric tabs, so it doesn't select anything when the flyout is opened there.

Reply:
Good catch 👍